### PR TITLE
Adapt the workflows to run against 1.0.x branch

### PR DESCRIPTION
**Pull request by violetagg**

### CODE REVIEW

**Comment by simonbasle** [comment]
@violetagg LGTM, understanding that we don't want to produce snapshots just yet for the `main` branch, correct? not until the `version` is changed to `1.1.0-SNAPSHOT` at least

**Comment by violetagg** [comment]
> @violetagg LGTM, understanding that we don't want to produce snapshots just yet for the `main` branch, correct? not until the `version` is changed to `1.1.0-SNAPSHOT` at least

I'm gonna change the version in main in a while :)

**Review by pderop — APPROVED** [review]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -154,7 +154,7 @@ spotless {
 		enforceCheck false
 	}
 	else {
-		String spotlessBranch = "origin/main"
+		String spotlessBranch = "origin/1.0.x"
```
Comment (simonbasle) [inline build.gradle:157]: NB: this one will need to be brought back to `main` once forward-merged

**Review by simonbasle — APPROVED** [review]

**Comment by violetagg** [comment]
@simonbasle @pderop Thanks for the review!

**Pull request merged**